chore: docs for the github app (#507)
bytebase/bytebase.com: content/docs/en/sql-review/sql-advisor/overview.md

- [**UI**](/docs/sql-review/sql-advisor/ui) - If you use Bytebase as a whole system, SQL Advisor will check common issues in schema change process;
- [**API**](/docs/sql-review/sql-advisor/api) - If you have your own database management system, it's possible to integrate SQL Advisor via API;

> - [**GitHub Action**](/docs/sql-review/sql-advisor/github-action) - If all your sql files are stored on GitHub.com, you can use GitHub action to do prechecks before they're merged into the main branch.

Reviewer: ```suggestion
- [**GitHub Action**](/docs/sql-review/sql-advisor/github-action) - If all your SQL files are stored on GitHub.com, you can use GitHub action to do prechecks before they're merged into the main branch.
```

> - [**GitHub App**](/docs/sql-review/sql-advisor/github-app) - If all your sql files are stored on GitHub.com, you can use GitHub app to do prechecks before they're merged into the main branch. No coding is required, just works in a few seconds of configuration in the UX.

Reviewer: ```suggestion
- [**GitHub App**](/docs/sql-review/sql-advisor/github-app) - If all your SQL files are stored on GitHub.com, you can use GitHub app to do prechecks before they're merged into the main branch. No coding is required, deploy your SQL check in minutes.
```
